### PR TITLE
feat: ACH drop-in support

### PR DESCRIPTION
**Pull request by FlaviuExtPrimer**

### CODE REVIEW

File: packages/sdk/android/src/main/java/com/primerioreactnative/extensions/PrimerPaymentMethodOptionsRN.kt [hunk 1]
```diff
@@ -27,4 +30,13 @@ fun PrimerKlarnaOptionsRN.toPrimerKlarnaOptions() =
...
+fun PrimerStripeOptionsRN.toPrimerStripeOptions(context: Context) = 
+    PrimerStripeOptions(mandateData = mandateData?.toMandateData(context), publishableKey = publishableKey)
+
+private fun PrimerStripeOptionsRN.MandateDataRN.toMandateData(context: Context) = when {
+    merchantName != null -> TemplateMandateData(merchantName)
+    fullMandateStringResName != null -> FullMandateData(
+        context.getResources().getIdentifier(fullMandateStringResName, "string", context.getPackageName())
+    )
+    else -> error("Missing mandate data")
+}
```
Comment (semirp) [inline packages/sdk/android/src/main/java/com/primerioreactnative/extensions/PrimerPaymentMethodOptionsRN.kt:42]: [nit]: new line

File: packages/sdk/src/models/PrimerSettings.ts [hunk 2]
```diff
@@ -150,4 +150,15 @@ interface IPrimerThreeDsOptions {
...
+
+interface IPrimerStripeMandateData {}
+
+interface IPrimerStripeTemplateMandateData extends IPrimerStripeMandateData {
+  merchantName: string;
+}
+
+interface IPrimerStripeFullMandateData extends IPrimerStripeMandateData {
+  fullMandateResourceKey: string;
 }
```
Comment (semirp) [inline packages/sdk/src/models/PrimerSettings.ts:164]: [nit]: new line

**Review by semirp — APPROVED** [review]

File: packages/sdk/android/src/main/java/com/primerioreactnative/extensions/PrimerPaymentMethodOptionsRN.kt (hunk 1 above)
Comment (FlaviuExtPrimer) [reply packages/sdk/android/src/main/java/com/primerioreactnative/extensions/PrimerPaymentMethodOptionsRN.kt]: Done.

File: packages/sdk/src/models/PrimerSettings.ts (hunk 2 above)
Comment (FlaviuExtPrimer) [reply packages/sdk/src/models/PrimerSettings.ts]: Done.

**Review by jnewc — APPROVED** [review]

**Review by semirp — APPROVED** [review]

**Pull request merged**